math/polycalc/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        Zeta611 polycalc 8c6922fab48f8e23ac4f02d00cd27cb271a02965
+version             2023.01.03
+revision            0
+categories          math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         PolyCalc: Polynomial Calculator
+long_description    PolyCalc is a polynomial calculator. It calculates and expands polynomials, equations and relations. \
+                    It supports simple variable assignments for the ease of handling expressions.
+checksums           rmd160  54221d07103237afc407a791e5d0b725fc2da7a9 \
+                    sha256  de11c920f403d8a87f9415d11df7eb8cdc212ea799f13b3b4f0f24bf31f6fc17 \
+                    size    12422
+
+depends_build-append \
+                    port:bison \
+                    port:flex
+
+compiler.c_standard 2017
+
+post-patch {
+    reinplace "s,gcc,${configure.cc}," ${worksrcpath}/Makefile
+}
```

Comment:
```suggestion
```

Generally, this sort of thing is better as a patch file. However, I don't think it's needed here. The makefile portgroup should add `CC="${configure.cc}"` as a build arg.

Reply:
You are right, this appears unneeded. I have removed it, and still correct GCC is chosen. I will delete this block from the portfile now.

Reply:
Worked for me, but fails on CI:
```
/opt/local/var/macports/build/_Users_runner_work_macports-ports_macports-ports_ports_math_polycalc/polycalc/work/compwrap/cc/usr/bin/clang -I./src -MMD -MP -O3 -Wall -Wextra -Wpedantic -std=c17 -Wno-implicit-function-declaration -c build/./src/poly.tab.c -o build/./src/poly.y.o
[362](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:365)
  src/poly.y:180:1: error: unknown type name 'near'
[363](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:366)
  near line %d\n", progname, msg, lineno);
[364](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:367)
  ^
[365](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:368)
  src/poly.y:180:10: error: expected ';' after top level declarator
[366](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:369)
  near line %d\n", progname, msg, lineno);
[367](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:370)
           ^
[368](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:371)
           ;
[369](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:372)
  src/poly.y:180:15: warning: missing terminating '"' character [-Winvalid-pp-token]
[370](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:373)
  near line %d\n", progname, msg, lineno);
[371](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:374)
                ^
[372](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:375)
  src/poly.y:182:1: error: extraneous closing brace ('}')
[373](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:376)
  }
[374](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:377)
  ^
[375](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:378)
  1 warning and 3 errors generated.
[376](https://github.com/macports/macports-ports/actions/runs/3838206817/jobs/6534389403#step:8:379)
  make: *** [build/./src/poly.y.o] Error 1
```